src/api/spec/factories/users.rb

```diff
@@ -2,7 +2,7 @@
   factory :user do
     email { Faker::Internet.email }
     realname { Faker::Name.name }
-    login { Faker::Internet.user_name(nil, %w(_)) }
+    sequence(:login){|n| "#{Faker::Internet.user_name(nil, %w(_))}#{n}" }
```

Comment:
@mdeniz It won't. But that's a different issue. If a test uses the backend, we need to ensure that the test always uses the same data for each test run to match with the VCR cassetes.
But IMO our factories should by default return valid objects, that are not identical.